userspace/libsinsp/container_engine/docker.h

```diff
@@ -22,9 +22,22 @@ limitations under the License.
 #include <memory>
 #include <string>
 #include <vector>
+#include <atomic>
+
+#if !defined(_WIN32)
+#include <curl/curl.h>
+#include <curl/easy.h>
+#include <curl/multi.h>
+#endif
+
+#include "tbb/concurrent_hash_map.h"
 
 #include "json/json.h"
 
+#include "async_key_value_source.h"
+
+#include "sinsp.h"
```

Comment:
would `class sinsp;` instead of including the whole world with this header work?

Reply:
I'm personally not-a-fan of individual class definitions, although I know that including sinsp.h does pull in the world. Someday I hope we can untangle the header dependencies so they include just what they need.

Reply:
Using `class sinsp;` and removing the `#include` would bring us one line closer to that goal :)

Reply:
Ok, I'll change it.